apps/src/code-studio/progressRedux.js

```diff
@@ -658,6 +661,83 @@ export const progressionsFromLevels = levels => {
   return progressions;
 };
 
+export const queryUserProgress = (userId, onComplete) => (
```

Comment:
This looks like an action creator. please put it with the other action creators: https://github.com/code-dot-org/code-dot-org/blob/ff90abb8c697c9ce7f294cd193e011f55fa3561d/apps/src/code-studio/progressRedux.js#L307